packages/create-breadboard-kit/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "wireit": {
     "copy-assets": {
-      "command": "cp -r assets dist",
+      "command": "mkdir -p dist/ && cp -r assets/ dist/assets/",
```

Comment:
The dist folder should already be created iirc - if it's not it suggests that there is an ordering of the build scripts issue... (also the trailing / is not needed.)